tests/zfs-tests/tests/functional/cli_root/zfs_destroy/zfs_destroy_common.kshlib

```diff
@@ -97,7 +97,7 @@ function setup_testenv #[dtst]
 #
 function cleanup_testenv
 {
-	if is_global_zone && ismounted "$TESTDIR1" "ufs" ; then
+	if is_global_zone && ! ismounted "$TESTDIR1" "ufs" ; then
```

Comment:
You should use NEWFS_DEFAULT_FS here to cleanup the local ext2 filesystem created in setup_testenv() with NEWFS.

```
-	if is_global_zone && ismounted "$TESTDIR1" "ufs" ; then
+-	if is_global_zone && ismounted "$TESTDIR1" "$NEWFS_DEFAULT_FS" ; then
```